paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -63,7 +62,7 @@ class PaparazziPlugin : Plugin<Project> {
   }
 
   private fun setupPaparazzi(project: Project) {
-    val unzipConfiguration = project.setupPlatformDataTransform()
+    val nativePlatformConfiguration = project.setupPlatformDataTransform()
```

Comment:
I'd rename setupPlatformDataTransform to getLayoutlibNative and split out the testImplementation dependency addition into a separate method since they serve very different purposes.

Return FileCollection instead of configuration and put the `.incoming.artifactView {}.files` inside of this method to hide away the implementation details.

Reply:
great suggestion.  done (with slightly different naming).